src/js/components/Dashboard/Overview/index.js

```diff
@@ -17,18 +24,24 @@ export default class extends Component {
   }
 
   render() {
-    const { wallets, coins } = this.props
-    return (
+    const { coins, wallets } = this.props
+    return coins.fetched && wallets.fetched ? ( // make sure app state contains what i need
       <ContentComponent
         header="Overview"
-        subHeader="Manage your assets at a glance"
+        subHeader="Manage your assets at a glance."
       >
-        {!coins.fetched || !wallets.fetched ? (
-          <Loader active />
-        ) : (
-          <HeaderStatistics wallets={wallets.by_name} coins={coins.by_symbol} />
-        )}
+        <link
+          rel="stylesheet"
+          href="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.css"
+        />
+        <div class="column" style={{ marginTop: "60px" }}>
+          <div class="ui sixteen column grid">
+            <PieComponent coins={coins} wallets={wallets} />
+          </div>
+        </div>
       </ContentComponent>
+    ) : (
+      <div />
```

Comment:
do you want a loader to display by default?
if so, you can import from semantic and use:
`<Loader active />`